basic/escrow/src/lib.rs

```diff
@@ -0,0 +1,136 @@
+use scrypto::prelude::*;
+
+blueprint! {
+    struct Escrow {
+        token_a: Vault,
+        token_b: Vault,
+        token_a_account: Account,
+        token_b_account: Account,
+        account_a_badge_ressource: ResourceDef,
+        account_b_badge_ressource: ResourceDef,
```

Comment:
ressource -> resource

Reply:
Thank you, my main language is french so sometimes I put two "s" instead of one :)